2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,19 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    let averagePrices = [];
+
+    for (const stock of CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS){
```

Comment:
indentation is off in multiple places in this loop. Keeping them properly aligned makes it easier to see the flow of the code. Your IDE can help you with that, most also contain a hotkey to auto-indent and fix other styling issues